templates/tfengine/recipes/resources.hcl

```diff
@@ -434,6 +448,74 @@ schema = {
         }
       }
     }
+    dns_zones = {
+      description = "https://github.com/terraform-google-modules/terraform-google-cloud-dns"
+      type        = "array"
+      items = {
+        type = "object"
+        additionalProperties = false
+        required = [
+          "name",
+          "domain",
+          "type",
```

Comment:
add record set as well as it's using `hcl` (mandatory) above.

Reply:
Done.